sdk/keyvault/service.projects

```diff
@@ -1,6 +0,0 @@
-<Project>
```

Comment:
We also need to remove https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/service.projects.

However we shouldn't remove either of them until we fix the reliability issue when running the tests in playback mode as well.

Reply:
Now that I merged your other PR please go ahead and remove the other service.projects file here as well.

Reply:
done